kubesignin/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Kubesignin chart for Kubernetes
 name: kubesignin
-version: 0.2.4
+version: 0.3.4
```

Comment:
I am a bit curious about the .4 in the version 🤔

Reply:
Yeah, the major version just got bumped 🤷